src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -73,6 +76,230 @@ public enum DocumentFlags
         AccessRemoved = 1 << 1
     }
 
+#if NET6_0_WINDOWS || NET6_0_WINDOWS10_0_17763_0 //Needs to specify TargetPlatform Version, just net6.0-windows/NET6_0_WINDOWS doesn't work as expected..
```

Comment:
Maybe you should define the former manually, since the latter is not a guarantee, especially on our internal build machines (are you planning to get an upgrade to 2022 internally?)

Reply:
FYI (https://github.com/dotnet/designs/blob/main/accepted/2020/net5/net5.md#design): So complicated.. Will need additional time to figure this out to apply to our project..

Reply:
That's basically why I don't see the need to rush to put in .NET 6.  Why not wait until we can deprecate the older runtimes?

Reply:
Created CBL-2631